### ISSUE TITLE
Allow absolute URLs in OmvDataSource

### ISSUE THREAD

**Issue by dbabbs** [heremaps/harp.gl#904]
Enable any absolute url to be passed in `OmvDataSource`. Currently, the developer is limited to a set of choices in [`OmvRestClient.ts`
](https://github.com/heremaps/harp.gl/blob/master/%40here/harp-omv-datasource/lib/OmvRestClient.ts). This means that, unless the developer builds harp.gl from github source and adds their own `APIFormat`, harp.gl does not let you use a custom vector tile source.

**Use case:**
As a developer with a custom vector data source of:
`https://my-base-url.com/vector-tiles/{z}/{x}/{y}.mvt`

I would like to easily visualize it in harp.gl without having to modify harp.gl source code.

**Requirements**
1. Pass absolute URL when creating `OmvDataSource`. No need to specify `APIFormat`.
2. Pass query parameters in key value format. These will be appended and aded onto the url request. (optional)
3. Pass headers to the request. These will be sent with the request

**Example implementation**
```javascript
const vector = new OmvDataSource({
   url: "https://my-base-url.com/vector-tiles/{z}/{x}/{y}.mvt",
   styleSetName: "tilezen",
   params: {
      "apikey": "adavadv",
      "param2": "berlin"
   }
});
```

(HARP-7392)

**Comment by dbabbs** [comment]
Number 2 (pass query parameters in key value format) has been implemented by: #903

**Comment by zbigg** [comment]
Implemented in #903
